modules/appeals_api/spec/docs/contestable_issues/v0/contestable_issues_v0_spec.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+require 'swagger_helper'
+require Rails.root.join('spec', 'rswag_override.rb').to_s
+
+require 'rails_helper'
+require AppealsApi::Engine.root.join('spec', 'spec_helper.rb')
+require AppealsApi::Engine.root.join('spec', 'docs', 'shared_docs_examples.rb')
+
+def swagger_doc
+  "modules/appeals_api/app/swagger/contestable_issues/v0/swagger#{DocHelpers.doc_suffix}.json"
+end
```

Comment:
I think it would reduce confusion a little if we could retire some of the `DocHelpers` methods we've been using to resolve correct environment values while sharing docs specs between decision reviews and the segmented APIs. I'm trying to use them minimally here in the hope that we can remove some after the segmented API docs are all split out this way.